alacritty/src/config/bell.rs

```diff
@@ -1,11 +1,96 @@
 use std::time::Duration;
 
+use log::error;
+use serde::{self, Deserialize, Deserializer};
+
 use alacritty_config_derive::ConfigDeserialize;
 
 use alacritty_terminal::config::Program;
-use alacritty_terminal::term::color::Rgb;
+use alacritty_terminal::index::Point;
+use alacritty_terminal::term::{color::Rgb, SizeInfo};
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+struct BellRadii {
+    /// Horizontal radius in units of cells.
+    pub horizontal: f32,
+    /// Vertical radius in units of cells.
+    pub vertical: f32,
+}
+
+pub struct Rect {
```

Comment:
This type feels a bit out-of-place here. It could be moved somewhere else and perhaps also be used inside `renderer::rects::RenderRect`?